common/src/types/wallet.rs

```diff
@@ -216,7 +216,7 @@ impl Wallet {
         new_private_shares.push(new_blinder_private_share);
 
         let (new_private_share, new_public_share) = create_wallet_shares_from_private(
-            self.clone().into(),
+            &self.clone().into(),
```

Comment:
nit: can we remove the `clone().into()` here?

Reply:
No, the type conversion is necessary and `Into::into` needs ownership